demisto_sdk/commands/common/native_image.py

```diff
@@ -80,6 +81,21 @@ def load(
         )
         return native_image_config_content
 
+    def get_native_image_reference(self, native_image) -> Union[str, None]:
+        """
+        Gets the docker reference of the given native image
+
+        Args:
+            native_image (str): native image (for example: 'native:8.1')
+
+        Returns: The docker ref
+        """
+        if native_image_obj := self.native_images.get(native_image):
+            return native_image_obj.docker_ref
+
+        else:  # desirable native image not in self.native_images
+            return None
```

Comment:
```suggestion
        if native_image_obj := self.native_images.get(native_image):
            return native_image_obj.docker_ref
```

by default, `None` will be returned if nothing is returned from a function, however for redability I would suggest:

```suggestion
        if native_image_obj := self.native_images.get(native_image):
            return native_image_obj.docker_ref
        return None
```

Reply:
Removed the else.